packages/components/dialog/src/Dialog.tsx

```diff
@@ -3,7 +3,28 @@ import { type ReactElement } from 'react'
 
 import { DialogProvider } from './DialogContext'
 
-export type DialogProps = RadixDialog.DialogProps
+export interface DialogProps {
```

Comment:
Not sure if we should do this. It's the only way to register the comment for each type we use from an external resource or library, but this way we're forced to rewrite the entire interface 😞

Reply:
I think we should. Only because Radix does not commit to providing JSDoc for their components props.

Their documentation is using static documentation and no JSDoc like us, so basically they write the descriptions twice. I think it's the only way for us to not be dependant on how they write/expose their component props.

Reply:
Thanks for your feedback!! Left it as it is, I only had to fix some type errors